### ISSUE TITLE
Make a Power Apps theme

### ISSUE THREAD

**Issue by pomfrida** [equinor/design-system#1228]
- [x] Ask Bulb Digital about MIT license
- [x] [Update README](https://github.com/equinor/eds-powerapps-poc/blob/main/README.md) with references to Bulb Digital
- [ ] Create Github actions for publishing to @pomfrida’s dev environment
- [x] Make @pomfrida admin in @vnys’ dev environment and vice versa
- [x] Install CoE Theme Editor in dev environments
- [ ] Create environment variables with EDS Tokens

**Comment by vnys** [comment]
Installed the Theme Editor in my own environment, have to do the same with @pomfrida’s

**Comment by vnys** [comment]
Closing this for now. To make a long story short: Up until this point there hasn’t been a good way to create a theme for Power Apps. @pomfrida and myself spent some time earlier this year to find out how we could do it, and which type of Power App we thought we should focus on – as there are three types: Model driven, Canvas and Portal. We decided to focus our efforts on Canvas apps, as that was the kind of apps the vast majority of makers built in Equinor.

A user by the name of Sancho Harker (iAm_ManCat) discovered a way to use variables in themes, which was picked up by Bulb Digital, who made [a template that we started modifying](https://github.com/equinor/eds-powerapps-poc#readme). Meanwhile the CAT team in Microsoft made [their own solution](https://github.com/microsoft/coe-starter-kit/tree/main/Theming) based on the concept that Sancho Harker created where they used Dataverse tables to store the themes. When I discovered that, I thought why not make a model driven app built on the Dataverse as a theme editor, and base that on the CAT Teams solution, which would then be used to create a theme for Canvas apps – and also extend that to meet our requirements in the EDS. But then as I realised the plan long term seems to be to merge model and canvas apps ([public preview announced recently](https://powerapps.microsoft.com/en-us/blog/custom-pages-for-converging-model-driven-apps-and-canvas-apps/)) we would have to also create a solution that works for model driven apps, which has very limited theming capabilities.

Another problem was what to do with the components (or controls in Power Apps terminology). Power Apps in Teams used modern controls built on Fluent UI, while traditional Canvas apps did not. So we ran the risk of theming classic controls without knowing when the modern controls would be rolled out for everyone. Then mid July Microsoft announced that [a solution for themes across all Power Apps](https://docs.microsoft.com/en-us/power-platform-release-plan/2021wave2/power-apps/support-themes-across-power-apps) will be available in public preview in August and in General Availability in October, which pretty much means whatever we roll out now will be obsolete before we know it. So the model driven app plan is put on hold until we know what this new theming concept is about, and how to connect it to Figma.

What we do have now is an Equinor branded version of the Bulb Digital Template, which we _can_ make available for download – if we think that would bring any value to anyone. I’ll discuss that with the Disp team as well, but it appears they’re going for a model driven or hybrid approach now so I don’t know at this point if there’s anything they can use in that template.

CC: @nilsml, @mimarz, @pomfrida
